_posts/2018-03-08-roadmap.md

```diff
@@ -0,0 +1,221 @@
+---
+layout: post
+title: "Rust's 2018 roadmap"
+author: "Aaron Turon"
+---
+
+# Rust’s 2018 roadmap
+
+Each year the Rust community [comes together][roadmap-process] to set out a
+roadmap. This year, in addition to the [survey], we put out
+a [call for blog posts][blog-2018] in December, which resulted
+in [100 blog posts][read-rust] written over the span of a few weeks. The end
+result is the recently-merged [2018 roadmap RFC][rfc].
+
+[roadmap-process]: https://github.com/rust-lang/rfcs/pull/1728
+[survey]: https://blog.rust-lang.org/2017/09/05/Rust-2017-Survey-Results.html
+[blog-2018]: https://blog.rust-lang.org/2018/01/03/new-years-rust-a-call-for-community-blogposts.html
+[read-rust]: https://readrust.net/rust-2018/
+[rfc]: https://github.com/rust-lang/rfcs/pull/2314
+
+## Entering the Rust 2018 epoch
+
+**This year, we will deliver _Rust 2018_, marking the first new era in Rust programming since 1.0**.
+
+We will continue to publish releases every six weeks as usual. But we will
+designate a release in the latter third of the year (Rust 1.29 - 1.31) as the
+beginning of the Rust 2018 epoch. This “epoch release” will be the culmination
```

Comment:
FWIW, I have been running the "era" wording past some people and they all vastly preferred it to "epoch"

Reply:
Interesting! I like "era" as well, because it more strongly conveys the idea of a period of time rather than a *point* in time. "epoch" can mean that in the archaeology sense, but because of the use of "epoch" in computing (January 1, 1970), there's also a feeling of "point in time" or "start of a new point in time" instead. "era" doesn't have that problem.